### PR TITLE
add typography Title.tsx and Text.tsx 

### PR DESCRIPTION
**Pull request by TurgunovAmir07**
image
image

### CODE REVIEW

File: src/components/ui/typography/Text.tsx
Comment (EduardDE7) [inline src/components/ui/typography/Text.tsx:1]: add nunito font, leading in px, remove  type className -  any, use in props className || ''

File: src/components/ui/typography/Title.tsx
Comment (EduardDE7) [inline src/components/ui/typography/Title.tsx:1]: add monserat font, remove size, size written together in the variant, add leading, remove type className - any, use in props className || ''

File: tailwind.config.ts
Comment (EduardDE7) [inline tailwind.config.ts:1]: remove font config

File: src/app/layout.tsx
Comment (EduardDE7) [inline src/app/layout.tsx:1]: remove inter

File: src/app/globals.css
Comment (EduardDE7) [inline src/app/globals.css:1]: remove inter, only 2 fonts

**Review by EduardDE7 — CHANGES_REQUESTED** [review]

**Review by EduardDE7 — DISMISSED** [review]

**Review by EduardDE7 — APPROVED** [review]

**Review by coldnanee — APPROVED** [review]

**Pull request merged**